services/asg.py

```diff
@@ -2,16 +2,40 @@
 import boto3
 
 
-# client = boto3.client("")
+client = boto3.client("autoscaling")
 
 
 def autoscaling_group_elb_healthcheck_required():
+    compliant_resources = []
+    non_compliant_resources = []
+    asgs = client.describe_auto_scaling_groups()["AutoScalingGroups"]
+
+    for asg in asgs:
+        if asg.get("HealthCheckType", {}) == "ELB":
+            compliant_resources.append(asg["AutoScalingGroupARN"])
+        else:
+            non_compliant_resources.append(asg["AutoScalingGroupARN"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resources,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def autoscaling_multiple_az():
+    compliant_resources = []
+    non_compliant_resources = []
+    asgs = client.describe_auto_scaling_groups()["AutoScalingGroups"]
+
+    for asg in asgs:
+        if len(asg.get("AvailabilityZones", 0)) > 1:
```

Comment:
ASG에는 항상 `AvailabilityZones` attribute가 존재합니다.
만약 attribute가 없어 0이 반환되더라도, `len()` 함수에 integer value가 들어가기 때문에 오류가 발생하게 됩니다.
```suggestion
        if len(asg["AvailabilityZones"]) > 1:
```